src/i18n/en/index.js

```diff
@@ -40,6 +40,9 @@ export default {
       importJSON: {
         title: 'Import QMK Keymap JSON file'
       },
+      importUrlJSON: {
+        title: 'Import QMK Keymap JSON from Url'
```

Comment:
`URL` should always be capitalized.

```suggestion
        title: 'Import QMK Keymap JSON from URL'
```

Reply:
Yeah, we left it unstyled because zekth isn't big on CSS. You want to take a stab at it @noroadsleft ? I was going to look at this weekend, but this is something right up your alley. I may write a better modal at a future date that can be re-used, but this is good enough for now mostly likely.

Reply:
Yeah, I've already got some ideas as far as CSS. It's not a lot of rules but it makes a big improvement visually IMO.